docs/spmd.md

```diff
@@ -202,6 +202,19 @@ The main use case for `XLAShardedTensor` [[RFC](https://github.com/pytorch/xla/i
 
 There is also an ongoing effort to integrate <code>XLAShardedTensor</code> into <code>DistributedTensor</code> API to support XLA backend [[RFC](https://github.com/pytorch/pytorch/issues/92909)].
 
+### DTensor Integration
+We are integrating PyTorch/XLA SPMD into DTensor API [RFC](https://github.com/pytorch/pytorch/issues/92909). We have a proof-of-concept integration for `distribute_tensor`, which calls `mark_sharding` annotation API to shard a tensor and its computation using XLA:
```

Comment:
are there any doc for distributed tensor we can link to in this doc?

Reply:
Good point, let me add this one? https://github.com/pytorch/pytorch/blob/main/torch/distributed/_tensor/README.md

Reply:
Done